blt/src/Blt/Plugin/Commands/AcquiaApi.php

```diff
@@ -0,0 +1,239 @@
+   *
+   * @param string $environment
+   *   Environment to effect.
+   * @param string $domain
+   *   Domain to add: foo.stanford.edu.
+   *
+   * @return bool|string
+   *   API Response.
+   */
+  public function addDomain($environment, $domain) {
+    return $this->callAcquiaApi("/environments/{$this->envIds[$environment]}/domains", 'POST', ['json' => ['hostname' => $domain]]);
+  }
+
+  /**
+   * Add a database to all environments.
+   *
+   * @param string $db_name
+   *   Database name to add.
+   *
+   * @return bool|string
+   *   API Response.
+   */
+  public function addDatabase($db_name) {
+    return $this->callAcquiaApi("/applications/{$this->envIds['appId']}/databases", 'POST', ['json' => ['name' => $db_name]]);
+  }
+
+  /**
+   * Add an SSL Certificate to a given environment.
+   *
+   * @param string $environment
+   *   Environment to effect.
+   * @param string $cert
+   *   SSL Cert file contents.
+   * @param string $key
+   *   SSL Key file contents.
+   * @param string $intermediate
+   *   SSL Intermediate certificate file contents.
+   * @param string|null $label
+   *   Label for Acquia dashboard.
+   *
+   * @return bool|string
+   *   API Response.
+   */
+  public function addCert($environment, $cert, $key, $intermediate, $label = NULL) {
+    if (is_null($label)) {
+      $label = date('Y-m-d G:i');
+    }
+    $data = [
+      'json' => [
+        'legacy' => FALSE,
+        'certificate' => $cert,
+        'private_key' => $key,
+        'ca_certificates' => $intermediate,
+        'label' => $label,
+      ],
+    ];
+    return $this->callAcquiaApi("/environments/{$this->envIds[$environment]}/ssl/certificates", 'POST', $data);
+  }
+
+  /**
+   * Activate an SSL cert already installed on the environment.
+   *
+   * @param string $environment
+   *   Environment to effect.
+   * @param int $certId
+   *   Certificate ID to activate.
+   *
+   * @return bool|string
+   *   API Response.
+   */
+  public function activateCert($environment, $certId) {
+    return $this->callAcquiaApi("/environments/{$this->envIds[$environment]}/ssl/certificates/{$certId}/actions/activate", 'POST');
+  }
+
+  /**
+   * Remove a cert from the environment.
+   *
+   * @param string $environment
+   *   Environment to effect.
+   * @param int $certId
+   *   Certificate ID to remove.
+   *
+   * @return bool|string
+   *   API Response.
+   */
+  public function removeCert($environment, $certId) {
+    return $this->callAcquiaApi("/environments/{$this->envIds[$environment]}/ssl/certificates/{$certId}", 'DELETE');
+  }
+
+  /**
+   * Get all SSL certs that are on the environment.
+   *
+   * @param string $environment
+   *   Environment to effect.
+   *
+   * @return bool|array
+   *   API Response.
+   */
+  public function getCerts($environment) {
+    if ($response = $this->callAcquiaApi("/environments/{$this->envIds[$environment]}/ssl/certificates")) {
+      return json_decode($response, TRUE);
+    }
+    return $response;
```

Comment:
This seems trixy. Wouldn't it be more clear if the last return is `return FALSE` instead of relying on the return from the method call?

Reply:
yeah i suppose it would be the same result. just easier to follow.